recipe/meta.yaml

```diff
@@ -13,26 +13,30 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 202
-  # NOTE: Conda forge lacks openblas on Windows, so skip the Windows build.
-  skip: true  # [win]
-  features:
-    - blas_{{ variant }}
+  number: 0
 
 requirements:
   build:
-    - {{ compiler('fortran') }}
+    - {{ compiler('fortran') }}  # [not win]
     - {{ compiler('c') }}
+    - flang  # [win]
+    # on Windows, this relies on having visual studio CE 2015
+    # this link needed quite some searching, please do not delete!
+    # https://go.microsoft.com/fwlink/?LinkId=532606&clcid=0x409
+
   host:
     - python
-    - blas 1.1 {{ variant }}
-    - openblas
     - numpy
+    - libflang  # [win]
+    - libgfortran  # [not win]
+    - lapack
+
   run:
-    - python
-    - blas 1.1 {{ variant }}
-    - openblas
     - {{ pin_compatible('numpy') }}
+    - lapack
+    - libgfortran  # [not win]
+    - libflang  # [win]
```

Comment:
Remove 2 lines above. Change lapack->openblas and keep `blas 1.1 {{variant}}` line